source/reference/minio-server/settings/core.rst

```diff
@@ -43,9 +43,21 @@ For example, to set up ftp access, you could set the variable to something like
 .. code-block:: shell
    :class: copyable
 
-   export MINIO_OPTS=' "--console-address="9001" --ftp="address=:8021" --ftp="passive-port-range=30000-40000" '
+   export MINIO_OPTS=' --console-address=":9001" --ftp="address=:8021" --ftp="passive-port-range=30000-40000" '
+
+On Unix-like systems using the recommended MinIO ``systemd`` service, you can save a file with the environment variable to ``/etc/default/minio`` instead of setting the variable manually. 
+
+.. note::
 
-On Unix-like systems, you can save a file with the environment variable to ``/etc/default/minio`` instead of setting the variable manually.
+   The MinIO server does not read ``MINIO_OPTS`` directly.
+   Instead, use the environment variable to pass options with variable expansion when starting the MinIO server.
+   
+   For example:
+   
+   .. code-block::
+      :class: copyable
+
+      minio server $MINIO_OPTS ...
```

Comment:
I think we're backing into what is important here, and readers do not tend to check too closely for details. Suggestion:

```suggestion
For Unix-like systems using the recommended MinIO ``systemd`` service, use the ``/etc/default/minio`` file and create an environment variable ``MINIO_OPTS`` for specifying command line parameters:

.. code-block:: shell
   :class: copyable

   MINIO_OPTS=' --console-address=":9001" --ftp="address=:8021" --ftp="passive-port-range=30000-40000" '

For systems running ``minio`` on the command line, you can pass parameters directly to the server *or* specify the environment variable as part of the command:

.. code-block:: shell
   :class: copyable

   export MINIO_OPTS=' --console-address=":9001" --ftp="address=:8021" --ftp="passive-port-range=30000-40000" '

   minio server $MINIO_OPTS ...

.. important::

   The ``minio`` server process does not read ``MINIO_OPTS`` directly.
   It only has meaning if used through MinIO's recommended ``minio.service`` file *or* by explicitly specifying the environment variable as part of commandline startup.
```

Reply:
Also not sure we want to mention `--ftp`` here. ``--certs-dir` might be more appropriate, ftp is a rarer case overall.

Reply:
<img width="615" alt="Screenshot 2024-06-21 at 13 10 07" src="https://github.com/minio/docs/assets/453297/159bd599-09ba-47a1-b66c-d509b56f459c">

Thanks @ravindk89, good suggestions, I cannot commit them through Github's UI for some reason (see above error). 

Could you please push those changes as a commit to this branch?

Reply:
I can't either - looks like a github issue. Let me see if I can do this in my local and force-push up, then I'll merge.